### PR TITLE
MRG: add list of MNE publications parsed from Google scholar

### PR DESCRIPTION
**Pull request by jasmainak**
closes https://github.com/mne-tools/mne-python/issues/1935

### CODE REVIEW

**Comment by jasmainak** [comment]
there isn't much I can do with Chinese characters. We agreed with @agramfort that this would be only run in "offline mode"

**Comment by Eric89GXL** [comment]
LGTM, I assume you've tested it and it works? Maybe add to the docstring at the top saying when it should be run, and what it outputs (and inputs it needs, if any)

**Comment by agramfort** [comment]
DO NOT MERGE

let's avoid putting a new file. These make packaging a pain.

Also put the generated .rst file in the repo and integrate it in the doc

**Comment by Eric89GXL** [comment]
I'm not sure what the output looks like, but it should have the date the citations were checked, so people know there might be newer ones if we don't update it for months

**Comment by jasmainak** [comment]
okay, I added the parsed `rst` file. Now just need to remove the `scholar_parser` for painless packaging.

**Comment by jasmainak** [comment]
@agramfort take a look now. I put it all in one file -- no dependencies. How does this look?

**Comment by jasmainak** [comment]
sorting by year isn't really working at the moment ... not sure why

File: doc/cited.rst [hunk 1]
```diff
@@ -0,0 +1,141 @@
+.. _cited
+
+Citations
```
Comment (agramfort) [inline doc/cited.rst]: Can you use a more explicit name?

Publications from MNE users

see as an example http://neuroimage.usc.edu/brainstorm/Pub

**Comment by jasmainak** [comment]
@agramfort title changed and now you have union of the citations for the two publications

**Comment by jasmainak** [comment]
oh wait, hold on. There is a weird misalignment ... will fix soon

**Comment by jasmainak** [comment]
ok fixed. Good to go from my end.

File: doc/cited.rst [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+102. `Related articles. </scholar?q=related:L3LMcU0_vPEJ:scholar.google.com/&hl=en&num=20&as_sdt=0,5&sciodt=0,5>`_. E Lau, E Nguyen.
+103. `VECTOR l0 LATENT-SPACE PRINCIPAL COMPONENT ANALYSIS. <http://www.mirlab.org/conference_papers/International_Conference/ICASSP%202014/papers/p4262-luessi.pdf>`_. M Luessi, MS Hämäläinen, V Solo - mirlab.org.
+104. `Tempo Estimation from the EEG signal during perception and imagination of music. <http://bib.sebastianstober.de/bcmi2015.pdf>`_. A Sternin,S Stober,AM Owen,JA Grahn- bib.sebastianstober.de.
+105. `Règles de sélection de variables pour accélerer la localisation de sources en MEG et EEG sous contrainte de parcimonie. <http://www.josephsalmon.eu/papers/gretsi2015.pdf>`_. O FERCOQ,A GRAMFORT,J SALMON- josephsalmon.eu.
+106. `MEG connectivity and power detections with Minimum Norm Estimates require different regularization parameters. <http://downloads.hindawi.com/journals/cin/aip/541897.pdf>`_. AS Hincapié, J Kujala, J Mattout, S Daligault… - downloads.hindawi.com.
+107. `E Halgren. </citations?user=eBqrsP0AAAAJ&hl=en&oi=sra>`_. K Giber, D Perruchoud,E Halgren, SS Cash - Nature.
```
Comment (agramfort) [inline doc/cited.rst]: this last one is weird

Comment (jasmainak) [inline doc/cited.rst]: I agree. But it's a problem with google scholar. See this: https://scholar.google.fr/scholar?q=giber+halgren&btnG=&hl=en&as_sdt=2005&sciodt=0%2C5&cites=12188330066413208874&scipsc=1. I can add a line in the code to remove it manually if you want.

Comment (agramfort) [inline doc/cited.rst]: just filter out the papers that don't have a url that starts with http or ftp

**Comment by jasmainak** [comment]
@agramfort publications without any urls filtered out

**Comment by Eric89GXL** [comment]
FYI I killed the CIs to save a bit of time

**Comment by jasmainak** [comment]
woops, sorry I should have used the skip-ci thing :)

**Comment by Eric89GXL** [comment]
Not a problem, it's a slow day anyway :)

File: doc/index.rst [hunk 3]
```diff
@@ -97,6 +97,7 @@ it is supported by the IDEX Paris-Saclay, ANR-11-IDEX-0003-02, via the
    getting_started
    whats_new
    cite
+   cited
    references
    tutorials
    auto_examples/index
```
Comment (agramfort) [inline doc/index.rst]: I would put it a the end of the list.

**Comment by agramfort** [comment]
@Eric89GXL merge if you're happy

File: doc/index.rst (hunk 3 above)
Comment (Eric89GXL) [inline doc/index.rst]: I'll end up moving it in my PR anyway

**Comment by Eric89GXL** [comment]
Thanks @jasmainak

**Pull request merged**